docs/usage/specific-behaviors.rst

```diff
@@ -0,0 +1,16 @@
+******************
+Specific Behaviors
+******************
+
+This page documents how our backend handles certain situations.
+
+
+``MACOSX_DEPLOYMENT_TARGET``
+============================
+
+
+The target MacOS version can by changed by setting the
+``MACOSX_DEPLOYMENT_TARGET`` environment variable.
+
+If ``MACOSX_DEPLOYMENT_TARGET`` is set, we will use the selected version for the
+wheel tag, otherwise ``platform.mac_ver()`` will be used instead.
```

Comment:
How about this clarification:
```
wheel tag, otherwise ``platform.mac_ver()`` (i.e., the current macOS version on
the build machine) will be used instead.
```